protocol/testing.go

```diff
@@ -264,8 +273,10 @@ func NewMockSubscription() *mockSubscription {
 
 func (s *mockSubscription) AppendBlocks(blocks []*types.Block) {
 	for _, b := range blocks {
+		status := HeaderToStatus(b.Header)
 		s.eventCh <- &blockchain.Event{
-			Difficulty: new(big.Int).SetUint64(b.Header.Difficulty),
+			// Difficulty: new(big.Int).SetUint64(b.Header.Difficulty),
```

Comment:
Is this leftover?

Reply:
[Fixed](https://github.com/0xPolygon/polygon-sdk/pull/208/commits/2bebf5039c1a5bc80ee7ff0639996cf8f84f0c33_